CalPatRec/fcl/prolog.fcl

```diff
@@ -350,31 +198,31 @@ CalPatRec : { @table::CalPatRec
             ]
         }
 
-        CalTrkFit          : {   module_type : KalFinalFit
-            diagLevel                                   : 0
-            debugLevel                                  : 0
-            printFrequency                              : 100
-            cprmode                                     : 1
-            addhits                                     : true
-            ComboHitCollection                          : makeSH
-            StrawDigiCollection                         : makeSD
-            StrawHitFlagCollection                      : "DeltaFinder:StrawHits"
-            SeedCollection                              : CalSeedFit
-            CaloClusterCollection                       : "CaloClusterMaker"
-            mcDigiCollTag                               : "dave,_it_is_used_only_when_debugging_MC._pasha"
-            DtMaxMiss                                   : 55.
-            MaxAddDoca                                  : 5.    # mm
-            MaxAddChi                                   : 5.    # 3.5 used by unweedHits and addHits
-            GoodKalSeedFitBits                          : ["SeedOK"]
-            fitparticle                                 : @local::Particle.eminus
-            fitdirection                                : @local::FitDir.downstream
-            KalFit                                      : { @table::CalPatRec.CprKFFinal }
-            AddHitBackgroundBits                        : ["Dead"]
-            diagPlugin : { tool_type  : "KalFinalFitDiag"
-                mcTruth       : 0
-                mcUtils       : { @table::TrkReco.McUtils }
-    }
-        }
+#         CalTrkFit          : {   module_type : KalFinalFit
```

Comment:
why comment out instead of remove?

Reply:
keeping things "on the table" makes it easier to remember what is left to be done...